src/plugin/components.js

```diff
@@ -1,27 +1,34 @@
 /**
  * Created by Vouill on 09/07/17.
  */
-import { vueBulmaDefaultRenderElement, camelCaseToDash } from './helpers'
+import { camelCaseToDash, isBulmaAttribute, getOutrEl } from './helpers'
 
 export const componentGenerator = (name, reqOuterElement) => ({
   name,
   functional: true,
-  render (h, {children, props, data}) {
-    const {outerElement, ...otherProps} = props
-    return h(outerElement || reqOuterElement || vueBulmaDefaultRenderElement.get(name) || 'div',
-      {
-        class: [camelCaseToDash(name), ...Object.keys(otherProps)
-          .filter(key => (
-            (otherProps[key] !== false)))
+  render (h, { children, props, data }) {
+    const { outerElement, ...otherProps } = props
+
+    /*
+     * Remove attribute only if it is from Bulma
+     */
+    if (data.hasOwnProperty('attrs')) {
```

Comment:
I see what you did here, however i am curious to what lead you to do it. Do you know any drawbacks of having the attrs with the bulma attributes ?

Reply:
I see _Bulma attributes_, not only in class property. That's all, i wrote it is not necessary :)
And no the answer - just confusing

![browser](https://i.imgur.com/jhGAcrY.png)
![editor](https://i.imgur.com/HNrvzma.png)

Reply:
Yup, i never saw having them in attrs being an issue since these components are strictly done for styling and for events passed by the 'on' attribute. Anyway let's go with it 👍 , the performance cost should be almost nothing.